src/System.Data.SqlClient/src/ApiCompatBaseline.net46.txt

```diff
@@ -0,0 +1,7 @@
+MembersMustExist : Member 'System.Data.SqlClient.SqlBulkCopy.WriteToServer(System.Data.Common.DbDataReader)' does not exist in the implementation but it does exist in the contract.
```

Comment:
All of these should be available in 4.6.  Are we using the right ref-assemblies?